### PR TITLE
#6314 Replace nodejs installation with conda.

### PR DESCRIPTION
**Pull request by michalgce**

### CODE REVIEW

**Comment by scottdraves** [comment]
it should install nodejs with conda, where is that?

**Comment by michalgce** [comment]
Opss, added.

File: docker/base/Dockerfile [hunk 1]
```diff
@@ -44,13 +46,10 @@ echo "deb https://dl.yarnpkg.com/debian/ stable main" | tee /etc/apt/sources.lis
...
-#    MiniConda   #
-##################
-RUN echo 'export PATH=/opt/conda/bin:$PATH' > /etc/profile.d/conda.sh && \
-    wget --quiet https://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh -O ~/miniconda.sh && \
-    /bin/bash ~/miniconda.sh -b -p /opt/conda && \
-    rm ~/miniconda.sh
+##########
+#  Node  #
+##########
+RUN conda install -y -c conda-forge nodejs
```
Comment (scottdraves) [inline docker/base/Dockerfile]: consolidate with the other conda install, it's best to minimize the number of Docker commands.
combine the apt-get and conda installs as much as possible.

Comment (scottdraves) [reply docker/base/Dockerfile]: https://docs.docker.com/engine/userguide/eng-image/dockerfile_best-practices 
"Minimize the number of layers"

Comment (michalgce) [reply docker/base/Dockerfile]: GTK Thanks!

**Pull request merged**